#! /bin/sh /usr/share/dpatch/dpatch-run
## 50_OCTAVEHOME-2.9.dpatch by Rafael Laboissiere <rafael@debian.org>
##
## DP: Correct value for OCTAVEHOME in Debian systems

@DPATCH@

--- octave2.1-2.1.73.orig/doc/conf.texi	2006-07-28 19:31:58.000000000 +0200
+++ octave2.1-2.1.73/doc/conf.texi	2006-08-18 22:14:48.441613384 +0200
@@ -3,5 +3,5 @@
 @c For copying conditions, see the file gpl.texi.
 
 @set VERSION 2.1.x
-@set OCTAVEHOME /usr/local
+@set OCTAVEHOME /usr
 @set TARGETHOSTTYPE i586-pc-linux-gnu
--- octave2.1-2.1.73.orig/doc/interpreter/conf.texi	2006-07-28 19:31:58.000000000 +0200
+++ octave2.1-2.1.73/doc/interpreter/conf.texi	2006-08-18 22:14:48.444612928 +0200
@@ -3,5 +3,5 @@
 @c For copying conditions, see the file gpl.texi.
 
 @set VERSION 2.1.x
-@set OCTAVEHOME /usr/local
+@set OCTAVEHOME /usr
 @set TARGETHOSTTYPE i586-pc-linux-gnu
--- octave2.1-2.1.73.orig/doc/liboctave/conf.texi	2006-07-28 19:31:58.000000000 +0200
+++ octave2.1-2.1.73/doc/liboctave/conf.texi	2006-08-18 22:14:48.447612472 +0200
@@ -3,5 +3,5 @@
 @c For copying conditions, see the file gpl.texi.
 
 @set VERSION 2.1.x
-@set OCTAVEHOME /usr/local
+@set OCTAVEHOME /usr
 @set TARGETHOSTTYPE i586-pc-linux-gnu
